lib/parser.js

```diff
@@ -62,7 +62,9 @@ function isFeedLink (originUrl, base, tagName, attrs) {
     var type = attrs.type || attrs.TYPE;
 
     if (tagName == 'link' && inc(contentTypes, type)) {
+      if (base != null) {
```

Comment:
I think the problem occurs when both `base` and `href` variables are empty (null, undefined or empty string). As in the line below `base ?` condition is already defensive against empty `base`, but not empty `href`.

So adding href check to if above like `tagName == 'link' && href && inc...` should solve this issue. Do you think there would be another case?